packages/plugin-constraints/sources/tauModule.ts

```diff
@@ -42,15 +43,14 @@ const tauModule = new pl.type.Module(`constraints`, {
     if (workspace == null)
       return;
 
-    const manifest: {[key: string]: any} = workspace.manifest.raw!;
+    const manifest: any = workspace.manifest.raw!;
 
     // Field is not present => this predicate can never match
-    if (!(fieldName.id in manifest))
-      return;
```

Comment:
This early return has been dropped, is it expected? Shouldn't it become `typeof value === 'undefined'`?

Reply:
Right, re-added using typeof check